addons/website_forum/views/website_forum_profile.xml

```diff
@@ -90,7 +113,23 @@
                 </div>
                 <div role="tabpanel" class="o_wforum_profile_tab tab-pane" id="answers">
                     <div class="mb-4">
-                        <h5 class="border-bottom pb-1">
+                        <h5 class="border-bottom pb-1 container">
+                            <t t-if="count_questions > 0">
+                                <t class="float-right" t-call="website.website_search_box_input">
+                                    <t t-set="_form_classes" t-valuef="form-inline float-right"/>
+                                    <t t-set="search_type" t-valuef="forums"/>
+                                    <t t-if="forum_id" t-set="action" t-value="'/forum/%s%s' % (forum_id, '/tag/%s/questions' % slug(tag)) if tag else ''"/>
+                                    <t t-else="" t-set="action" t-value="'/forum/all%s' % ('/tag/%s/questions' % slug(tag)) if tag else ''"/>
+                                    <t t-set="display_description" t-valuef="true"/>
+                                    <t t-set="display_detail" t-valuef="true"/>
```

Comment:
I think here you should use `t-value` (here it will work because it contains the string "true' instead of the boolean)